cdc/processor.go

```diff
@@ -745,6 +750,9 @@ func (p *processor) syncResolved(ctx context.Context) error {
 			}
 			rows = append(rows, ev.Row)
 		}
+		failpoint.Inject("processorPanic", func() {
+			panic("processorPanic failpoint")
+		})
```

Comment:
We don't need to write panic explicitly, instead, please use `processorPanic ="1*panic"`  https://github.com/pingcap/failpoint#key-concepts